lib/api.js

```diff
@@ -311,7 +311,7 @@ class Api extends Emittery {
 	}
 
 	async _computeForkExecArgv() {
-		const execArgv = this.options.testOnlyExecArgv || process.execArgv;
+		const execArgv = this.options.nodeArguments;
```

Comment:
I think `testOnlyExecArgv` was used by AVA's internal tests, though I don't see a reference to it anymore. So removing that is fine.

`process.execArgv` would inherit the arguments used when starting AVA itself. I think we should concatenate that with `nodeArguments` more clearly *here*, rather than through a separate module.